leetduel-frontend/src/app/page.tsx

```diff
@@ -4,6 +4,12 @@ import { useRouter, useSearchParams } from "next/navigation";
 import socket from "../socket";
 import { useGame } from "../context/GameContext";
 
+enum PartyStatus {
+  UNJOINED = "unjoined",
+  JOINED = "joined",
+  CREATED = "created"
+}
+
```

Comment:
- `UNJOINED`: for before joining or creating a room
- `JOINED`: for joining another user's room
- `CREATED`: for creating a room. you are the host if you have this status